deps/+Boost/Boost.patch

```diff
@@ -0,0 +1,29 @@
+--- ./libs/json/include/boost/json/detail/charconv/detail/emulated128-orig.hpp	2024-07-16 21:55:14.679931600 -0600
++++ ./libs/json/include/boost/json/detail/charconv/detail/emulated128.hpp	2024-07-16 21:56:13.328860900 -0600
+@@ -58,7 +58,7 @@
+ 
+ static inline std::uint64_t umul64(std::uint32_t x, std::uint32_t y) noexcept
+ {
+-#if defined(BOOST_JSON_HAS_MSVC_32BIT_INTRINSICS)
++#if defined(BOOST_JSON_HAS_MSVC_32BIT_INTRINSICS) && !defined(_M_ARM64)
```

Comment:
@SachCZ I'm finally back. When do you think this will happen. I can rebase this branch (I'm 280+ commits behind). But I'd like to keep the branch in a buildable state